src/content/getting-started/observability/logging/disable/index.md

```diff
@@ -0,0 +1,74 @@
+---
+linkTitle: Disable logging
+title: Disable Logging
+description: This article explains how to turn off all logging for an entire workload cluster.
+weight: 50
+menu:
+  main:
+    identifier: getting-started-observability-logging-disable
+    parent: getting-started-observability-logging
+user_questions:
+  - How can I disable logging on my cluster?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2023-10-11
+aliases:
+  - /ui-api/observability/logging/disable
+  - /observability/logging/disable
+---
+
+In this article you will learn how you can disable logging for your cluster.
+
+## Introduction to logging
+
+Since releases 19.1.0, cluster created on the Giant Swarm platform benefits from our logging stack which allow us to provide you with 24/7 support to ensure best quality of service.
+
+Each components logs deployed in the `kube-system` and `giantswarm` namespaces as well as kubernetes and machine audit logs are collected by our managed `promtail` pods and sent to a Loki instance running in your management cluster. You can access its logs by accessing our managed Grafana.
```

Comment:
Is this extensible to other namespaces? Otherwise the stack isn't that helpful for customers.

Reply:
Not right now because this is only for Giant Swarm but will eventually (depending on the customer choice) take all theirs appications logs